src/libraries/MathLib.sol

```diff
@@ -10,11 +10,13 @@ int256 constant WAD_INT = int256(WAD);
 /// @custom:contact security@morpho.org
 /// @notice Library to manage fixed-point arithmetic on signed integers.
 library MathLib {
-    function wMulDown(int256 a, int256 b) internal pure returns (int256) {
+    /// @dev Returns the multiplication of `a` by `b` (in WAD) rounded towards 0.
+    function wMulTo0(int256 a, int256 b) internal pure returns (int256) {
         return a * b / WAD_INT;
     }
 
-    function wDivDown(int256 a, int256 b) internal pure returns (int256) {
+    /// @dev Returns the division of `a` by `b` (in WAD) rounded towards 0.
+    function wDivTo0(int256 a, int256 b) internal pure returns (int256) {
         return a * WAD_INT / b;
```

Comment:
Nitpick: follow the same style and names used by Morpho Blue `MathLib` contract

- `a * WAD_INT / b`→ `(a * WAD_INT) / b`
- `a` → `x` and `b` → `y`
- update the natspec documentation reflecting the changes (always following the MB natspec)